styles.css

```diff
@@ -59,21 +106,49 @@
   letter-spacing: 0;
   text-align: left;
   color: #7f8cff;
-  margin-left: 20px;
   padding-top: 15px;
 }
 
-.getresume {
-  margin-left: 20px;
-  padding-top: 15px;
-  padding-bottom: 30px;
+.about h3 {
+  padding-bottom: 20px;
+}
+
+.about-section {
+  display: flex;
+  flex-direction: column;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 104px;
+  min-width: 320px;
+  height: auto;
+}
```

Comment:
This section is not following the template design. You're missing the `border-radius`, you're missing the dividers between the technology categories, the margins and the padding are incorrect and your button doesn't look like that from the template design. Please fix them.